internal/ui/login/static/resources/themes/scss/variables.scss

```diff
@@ -33,11 +33,15 @@ $inputBackgroundColor: #252525;
 $buttonBackgroundColorHover: $inputBackgroundColor;
 $profileImgDark: "../../../images/icon-user-dark";
 $otherUserImgDark: "../../../images/icon-newuser-dark";
+$idpGoogleImg: "../../../images/idp/google2";
```

Comment:
there are 3 google logos uploaded in this PR but only 1 is referenced here

Reply:
removed 2 of them